types/koa-redis/index.d.ts

```diff
@@ -0,0 +1,21 @@
+// Type definitions for koa-redis 3.0
+// Project: https://github.com/koajs/koa-redis
+// Definitions by: Nick Simmons <https://github.com/nsimmons>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import {SessionStore} from 'koa-generic-session';
+
+declare namespace redisStore {
+    interface RedisOptions {
+        url?: string;
+        host?: string;
+        port?: number;
+        path?: string;
+        db?: number;
+        duplicate?: boolean;
+        client?: any;
+    }
```

Comment:
I think this should extend from "redis" module's ClientOpts instead of copying few fields from there to here.